Description: Raise the timeout for test commands to 10 minutes
 Some of the tests, e.g. the window-resize one, may take a long time on
 busy builder hosts.
Bug-Debian: https://bugs.debian.org/1032592
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2023-03-18

--- a/tests/cli-tests/run.py
+++ b/tests/cli-tests/run.py
@@ -639,7 +639,7 @@
         help="Preserve the scratch directory TEST_DIR/scratch/ for debugging purposes."
     )
     parser.add_argument("--verbose", action="store_true", help="Verbose test output.")
-    parser.add_argument("--timeout", default=60, type=int, help="Test case timeout in seconds. Set to 0 to disable timeouts.")
+    parser.add_argument("--timeout", default=60 * 10, type=int, help="Test case timeout in seconds. Set to 0 to disable timeouts.")
     parser.add_argument(
         "--exec-prefix",
         default=None,
